tests/nodeos_snapshot_diff_test.py

```diff
@@ -120,12 +120,13 @@ def waitForBlock(node, blockNum, blockType=BlockType.head, timeout=None, reportI
     waitForBlock(node0, blockNum, blockType=BlockType.lib)
 
     Print("Configure and launch txn generators")
-    producerP2pPort = cluster.getNodeP2pPort(snapshotNodeId)
     targetTpsPerGenerator = 667
```

Comment:
This is way overkill, 10 should be more than enough. Just need a few.

Reply:
Happy to reduce it. Original aim was to maintain generation rates original attempted when using the trx_test_gen_plugin.

Reply:
I created this test, no idea why I had it that large. Probably a copy/paste from somewhere else. Only need a couple of trx per block.

Reply:
Addressed: [de3a1c8](https://github.com/AntelopeIO/leap/pull/639/commits/de3a1c8392564d6d0ab41412374948c4c8f5359c)